README.md

```diff
@@ -1 +1,184 @@
+```
+Kindly paste your IP camera URL into your browser to confirm it is loading a live feed before feeding into DeepStack-CameraUI.
+
+* **Custom Model Name**
+
+This widget accepts the name of your custom model, it is optional only required when **custom detection** is selected.
+
+* **Start**
+
+This button should be clicked on to begin the program execution.
+
+* **Webhook URL**
+
+The URL inputted will receive the detailed informations about the content of the detections being captured in real-time and the raw input video frames as base64.
+
+* **Webhook Api Key**
+
+The definition here
+
+* **Webhook Forward Images**
+
+Here you can decide to receive the images of the live detection through the webhook URL or not.
+
+# DeepStack-CameraUI Techniques
+DeepStack-CameraUI delivers real-time detection through these six different techniques.
+
+### Object detection
+The object detection API locates and classifies 80 different kinds of objects in a live streaming.
+
+To use this API, you need to enable the detection API when starting DeepStack with docker through this command
+
+```
+docker run -e VISION-DETECTION=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face detection
+The face detection API aims at identifying a face in the real-time live streaming.
+To use this API, you need to enable the detection API when starting DeepStack with docker through this command
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face Registration
+The face Registration API goal's is to allow you to enroll a person's name and image on Deepstack server for future use like face recognition. The face recognition API is dependent of this API.
+
+To use this API, you need to enable the face API when starting DeepStack with docker through this command
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face Deregistration
+The face deregistration API goal's is to allow you to erase a person's name and image from Deepstack server. 
+
+To use this API, you need to enable the face API when starting DeepStack with docker through this command
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face Recognition
+The face recognition API goal's is to identify a face that has been previously register on Deepstack server, when it fails to recognize a face it portaits it as an **Unknown** face in the live stream.
+To use this API, you need to enable the face API when starting DeepStack with docker through this command
+
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Custom detection
+The custom detection API allows you to perform detection on personal/ specific object. To acheive this, your must prepare your own dataset, trained model , deplayed model and store these elements into a folder.
```

Comment:
You don't need to add the part `To acheive this, your must prepare your own dataset, trained model , deplayed model and store these elements into a folder`

Simply add a link to learn more about training custom model or using samples form the custom models samples page in the documentation